kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeploymentDependentResource.java

```diff
@@ -107,11 +123,17 @@ public static Map<String, String> podLabels(KafkaProxy primary) {
     private PodTemplateSpec podTemplate(KafkaProxy primary,
                                         KafkaProxyContext kafkaProxyContext,
                                         ProxyIngressModel ingressModel,
-                                        List<VirtualKafkaCluster> virtualKafkaClusters) {
-        // @formatter:off
-        return new PodTemplateSpecBuilder()
+                                        List<VirtualKafkaCluster> virtualKafkaClusters,
+                                        String checksum) {
+        PodTemplateSpecFluent<PodTemplateSpecBuilder>.MetadataNested<PodTemplateSpecBuilder> metadataBuilder = new PodTemplateSpecBuilder()
                 .editOrNewMetadata()
-                    .addToLabels(podLabels(primary))
+                .addToLabels(podLabels(primary));
+        if (!checksum.isBlank()) {
```

Comment:
same as other comment, should we compare to the constant if we're going to use blank as a special value?

Reply:
yeah, lets just have the conversation there.